eng/common/scripts/SemVer.ps1

```diff
@@ -14,8 +22,21 @@ class SemVer {
     [bool] $IsPrerelease
     [string] $RawVersion
 
-    SemVer(
+    AzureEngSemanticVersion(
```

Comment:
nit: I'd put the parameters on the same line `AzureEngSemanticVersion([string] $versionString)`

Reply:
Moved to one line when there is just a single parameter